docs/concepts/notifications.md

```diff
@@ -161,6 +161,8 @@ slack_notification = mlrun.model.Notification(
     secret_params={"secret": "SLACK_SECRET1"},
 )
 ```
+> ℹ️ **Info:** On running notification you can set only the webhook, and the message will be the default message.
```

Comment:
### "running" notifications 
"running" notifications have additional considerations:
- You can set only the webhook; the message is the default message.
- If the webhook is stored in the `secret_params`, you should first set the project secret and then use this project secret in the notification. For example:
...